src/native/_auth.js

```diff
@@ -214,7 +213,8 @@ module.exports = {
     access_container_fetch: helpers.Promisified(null, [ref.refType(ContainerPermissions), t.usize], (args) => {
       const ptr = args[0];
       const len = args[1];
-      let arrPtr = ref.reinterpret(ptr, ContainerPermissions.size * len);
+      if(len === 0) return [];
```

Comment:
This is wrong, you are returning an array when the function normally returns an object.